### ISSUE TITLE
Pairing blocked by popup blocker because of delay

### ISSUE THREAD

**Issue by klassare** [airgap-it/beacon-sdk#483]
**Describe the bug (current behavior)**
Sometime when clicking Kukai in the wallet selection modal, a loader will go off. After a couple of seconds it stops and a new tab is opened but blocked by popup blocker.

**To Reproduce**
- not easily reproducible

**Expected (correct) behavior**
There should never be a delay between the click and opening of the new tab. Tabs need to be opened as a direct consequence of a user action for them to not be blocked in every browser basically.

**Screenshots and/or logs**
![Skärmavbild 2023-04-19 kl  15 12 54](https://user-images.githubusercontent.com/31806893/234559872-0a4b919a-147b-4165-b0df-9313178d7ac9.png)

**Environment**

- Device: MacBook Pro
 - OS: macOS Ventura
 - Browser: Chrome

**Additional context**
Beacon version: 4.0.1-beta-kukai-wc2.0

**Comment by mlaeng** [comment]
This should have been resolved in this beta version 4.0.6-beta.0, see: https://www.npmjs.com/package/@airgap/beacon-sdk/v/4.0.6-beta.0

**Comment by jsamol** [comment]
As already mentioned, the issue should be resolved in the beta version. In case it's not, please open a new issue providing new information.

**Comment by klassare** [comment]
Can we keep this issue open until it is resolved in production?
